lib/stdext.mli

```diff
@@ -25,10 +25,10 @@ module Path : sig
       - ends with ['~']
       - or begins with ['#'] and ends with ['#']. *)
 
-  val find_files : name_wo_ext:string -> Fpath.t list -> Fpath.t list
+  val find_files : name_wo_ext:string list -> Fpath.t list -> Fpath.t list
   (** [find_files ~name_wo_ext files] returns the list of files among [files]
-      whose name without the extension is equal to [name_wo_ext]. Backup files
-      are ignored. *)
+      whose name without the extension is equal to an elemet of [name_wo_ext].
```

Comment:
there is a typo

Reply:
Well spotted. I also removed the "the" (which I think is more correct as there is no prior determination of the extension).